datafusion/sql/src/statement.rs

```diff
@@ -214,11 +215,13 @@ impl<S: ContextProvider> SqlToRel<'_, S> {
                 verbose,
                 statement,
                 analyze,
-                format: _,
+                format,
                 describe_alias: _,
                 ..
             } => {
-                self.explain_to_plan(verbose, analyze, DFStatement::Statement(statement))
+                let format = format.map(|format| format.to_string());
```

Comment:
Not very related to this PR, will we change sqlparser part to use a `String` instead of enum like here? (I'd prefer so, as it gives us more flexibility)